recipes/concoct/meta.yaml

```diff
@@ -1,15 +1,15 @@
 package:
   name: concoct
-  version: "0.4.2"
+  version: "1.0.0"
 
 source:
-  url: https://github.com/BinPro/CONCOCT/archive/0.4.2.tar.gz
-  sha256: 1242460a6d0b53e2aa28a1426213db397f6fd7dcaccd783cf24a2d09c035605c
+  url: https://github.com/BinPro/CONCOCT/archive/1.0.0.tar.gz
+  sha256: 4428a1d7bce5c7546106bea0a0ff6c2168e3680f6422adce5ef523bce46dc180
 
 build:
   preserve_egg_dir: True
-  number: 0
-  skip: True # [not py27]
+  number: 1
```

Comment:
Please start at 0 for new versions

Reply:
Ah, second time you have to correct me on this point now. This time I thought I did the right thing though. I thought I had to retrigger the tests by updating the build...

Reply:
You only need to bump the build number if the package was actually built and uploaded (i.e. merged into master).